back/backend/models/champion.py

```diff
@@ -0,0 +1,20 @@
+from google.cloud import ndb
+
+from backend.models.championship import Championship
+from backend.models.wca.event import Event
+from backend.models.wca.result import Result
+
+
+class Champion(ndb.Model):
+    championship = ndb.KeyProperty(kind=Championship)
+    event = ndb.KeyProperty(kind=Event)
+    champions = ndb.KeyProperty(kind=Result, repeated=True)
+
+    national_champion = ndb.ComputedProperty(lambda e: e.championship.get().national_championship)
```

Comment:
Was this supposed to be

```suggestion
    national_championship = ndb.ComputedProperty(lambda e: e.championship.get().national_championship)
```

?

Reply:
Additionally, should the entire thing be renamed to `is_national_championship` since it's a bool?

Reply:
It's in the champion object, not the championship. I agree with the `is_` though.